### DISCUSSION TITLE
절대경로 설정 관련 의견

### DISCUSSION THREAD

**Discussion by devkyoung2** [KakaoFunding/front-end#27] [General]
절대경로 설정 관련해서 두 개의 대안이 있는데 아직 확실하게 결정되지 않는 것 같아서 최종으로 의견 듣고 확정하려고 디스커션 남깁니다. 아래의 링크는 제가 예전에 디스코드에 남겼던 포스트에요

[React에서 컴포넌트 경로 깔끔히 관리하기](https://velog.io/@rimo09/React%EC%97%90%EC%84%9C-%EC%BB%B4%ED%8F%AC%EB%84%8C%ED%8A%B8-%EA%B2%BD%EB%A1%9C-%EA%B9%94%EB%81%94%ED%9E%88-%EA%B4%80%EB%A6%AC%ED%95%98%EA%B8%B0)

### 1. 절대경로 지정하기
tsconfig와 vite.config의 컴파일 옵션에서 진입점과 baseUrl 절대경로를 지정해주는 방법.

### 2. 모듈 다시 내보내기
각 요소를 디렉토리의 index.tsx 파일에서 다시 내보내주는 방법

사실 vite에서 ts를 js로 변환하는 과정에서 경로를 못찾아주는건지 계속 오류가 나네요 ㅠ.ㅠ
어제 명석님께서 보내주신 링크대로 적용했는데도 에러가 안잡혀서 2번 방법으로 하려고 하는데 다들 괜찮으실까요 ?
괜찮으시다면 따봉 남겨주세요  . . . . .

**Comment by uraflower** [comment]
보경님 혹시 그냥 지금까지 세팅하신거 main에 푸시해주실 수는 없나요?
바로 개발 시작하자는 게 아니라, 코드를 좀 보고 수정하면서 문제 해결을 좀 하고 싶어서요.
세 명이 생각을 한 다음 취합해서 한 명이 혼자 바꾸는 것보다 효율적일 것 같은데 어떠신가요?

**Reply by devkyoung2** [reply]
넵 푸시하고 디코로 멘션 드릴게요!
